rotkehlchen/chain/ethereum/modules/eth2/utils.py

```diff
@@ -257,67 +242,12 @@ def scrape_validator_daily_stats(
                     pnl -= INITIAL_ETH_DEPOSIT
 
                 column_pos += 1
-            elif column_pos == 3:
-                start_amount = _parse_fval(column.string, 'start amount')
-                column_pos += 1
-            elif column_pos == 4:
```

Comment:
wonder why we have this column pos instead of an enumerate

Reply:
Hmm don't know. Maybe @LefterisJP you know?

Reply:
dirty code written bit by bit to make the scraping parser work and then left as it is. If it still works you can ofc us an enumerate.

Reply:
Ah, I think it's the way it is due to:

```
if column.name != 'td':
                continue
```

So `column_pos` doesn't increment on every loop iteration